source/class/qx/tool/cli/Cli.js

```diff
@@ -170,7 +170,16 @@ qx.Class.define("qx.tool.cli.Cli", {
         .argv;
       await this.__notifyLibraries();
     },
-    
+
+    /**
+     * This is to notify the commands after loading the full args.
+     * The commands can overload special arg arguments here.
+     * e.g. Deploy will will overload the target.
+     */
+    __notifyCommand: function() {
+      this._compilerApi.getCommand().processArgs(this.argv);
+    },
+
     /**
```

Comment:
I don't know why this is in the PR, I didn't add it - it was in the source before!